atom/renderer/api/atom_api_spell_check_client.h

```diff
@@ -79,9 +85,8 @@ class SpellCheckClient : public blink::WebSpellCheckPanelHostClient,
   SpellcheckWordIterator text_iterator_;
   SpellcheckWordIterator contraction_iterator_;
 
-  bool auto_spell_correct_turned_on_;
-
   v8::Isolate* isolate_;
+  v8::Persistent<v8::Context> context_;
```

Comment:
I believe `mate::ScopedPersistent` can be generalized to handle this one too.

Reply:
Yeah, I didn't want to go that route just for this one variable. Also I didn't find `mate::ScopedPersistent` that useful an abstraction, but maybe I'm missing something.